resources/shaders/dxr_shadow_entries.hlsl

```diff
@@ -0,0 +1,26 @@
+#ifndef __RT_SHADOW_ENTRIES__
+#define __RT_SHADOW_ENTRIES__
+
+#include "pbr_util.hlsl"
+#include "material_util.hlsl"
+
+// Definitions for: 
+// - Vertex, Material, Offset
+// - Ray, RayCone, ReflectionHitInfo
+#include "dxr_structs.hlsl"
+
+[shader("closesthit")]
+void ShadowClosestHitEntry(inout ShadowHitInfo hit, Attributes bary)
+{
+	hit.is_hit = true;
+}
+
+[shader("miss")]
+void ShadowMissEntry(inout ShadowHitInfo hit : SV_RayPayload)
+{
+	hit.is_hit = false;
+}
+
+//TODO: Add any hit shader
```

Comment:
:O did you remove transparency?

Reply:
Not this pull request, that is for another branch

Reply:
This pull request is to prepare the ground for that as well